### PR TITLE
Update proto compilation and convert Long types to Number

### PR DESCRIPTION
**Pull request by adarob**

### CODE REVIEW

File: music/compile-proto.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Regenerates the protobuf
+#
+# To run, execute 'yarn proto'.
+
+# Clone magenta repo to get proto file
+git clone https://github.com/tensorflow/magenta.git
```
Comment (cghawthorne) [inline music/compile-proto.sh]: should this be cloned to a temp dir and then deleted after proto generation?

File: music/compile-proto.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Compile js
+yarn pbjs --force-number -t static-module -w commonjs -o src/protobuf/proto.js magenta/magenta/protobuf/music.proto
+# Compile ts
+yarn pbts -o src/protobuf/proto.d.ts src/protobuf/proto.js
+# Replace Long types with number
```
Comment (cghawthorne) [inline music/compile-proto.sh]: Reference protojs bug to hopefully eventually fix this?

Comment (adarob) [reply music/compile-proto.sh]: Done

File: music/compile-proto.sh (hunk 1 above)
Comment (adarob) [reply music/compile-proto.sh]: Done

**Comment by adarob** [comment]
FYI, this change was required to get around https://github.com/dcodeIO/protobuf.js/issues/1109

**Pull request merged**